packages/x-components/src/x-modules/ai/store/module.ts

```diff
@@ -60,6 +62,9 @@ export const aiXStoreModule: AiXStoreModule = {
     setParams(state, params) {
       state.params = params
     },
+    setOrigin(state, origin = null) {
```

Comment:
The origin payload is required to execute this mutation. Default value is not needed